css/index.js

```diff
@@ -249,7 +249,7 @@ module.exports = {
     'property-no-vendor-prefix': true,
     'rule-empty-line-before': null,
     'selector-attribute-quotes': 'always',
-    'selector-class-pattern': '^[a-z][a-z0-9\\-]*[a-z0-9]$',
```

Comment:
`\\` means the literal `\`. Is that even allowed in selector names?

Reply:
As far as I know classes should just not start with a number, but anything else is valid (some characters need to be escaped though).

Reply:
Well, if we want to allow backslashes, then this patch is wrong.